common/djangoapps/course_modes/views.py

```diff
@@ -190,11 +192,22 @@ def get(self, request, course_id, error=None):
                 for x in verified_mode.suggested_prices.split(",")
                 if x.strip()
             ]
+            min_price = verified_mode.min_price
+
             context["currency"] = verified_mode.currency.upper()
-            context["min_price"] = verified_mode.min_price
+            context["min_price"] = min_price
             context["verified_name"] = verified_mode.name
             context["verified_description"] = verified_mode.description
 
+            offer_banner_fragment = get_first_purchase_offer_banner_fragment(
+                request.user, course
+            )
+            if offer_banner_fragment:
```

Comment:
Do we write tests for views? If so, should we add them for this?

Reply:
yeah i'll add one

Reply:
@crice100 added test